client/src/Views/ReportsPage/AllShouts.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-tab-item class="all-container">
+  <div class="all-container">
```

Comment:
I don't think that this needed to have changed...seems to still work with v-tab-item

Reply:
For me, the routing doesn't work if I leave it as v-tab-item. Is that the better way to implement this?